src/app.js

```diff
@@ -1 +1,54 @@
-console.log('Hallo daar!');
\ No newline at end of file
+console.log("lorem");
+import axios from "axios";
+
+async function countryInfoFunction() {
+  try {
+    const countryInfoResult = await axios.get(
+      "https://restcountries.com/v2/all"
+    );
+
+    const countryNames = document.getElementById("countryDiv");
+
+    const sorter = countryInfoResult.data;
+    sorter.sort((a, b) => {
+      return a.population - b.population;
+    });
+
+    countryNames.innerHTML = countryInfoResult.data
+      .map((country) => {
+        return `
+
+<div class="flex">
+<li id="countryItem">
+<img src="${country.flags.svg}" alt="country flag" class="flag"/>
+<p class="${regionColorFunction(country.region)}"> ${country.name}</p> 
+<p>Total population of: ${country.population}</p>
+<p>located in: ${country.region}.</p>
+</li>
```

Comment:
Kleine opmerking bij dit stukje HTML die je gaat injecteren; de leesbaarheid zou hier iets groter zijn als het iets netter geformateerd is.